web3_proxy/src/stats/mod.rs

```diff
@@ -377,6 +599,13 @@ impl RpcQueryStats {
         let response_millis = metadata.start_instant.elapsed().as_millis() as u64;
         let response_bytes = response_bytes as u64;
 
+        // TODO: Gotta make the arithmetic here
+
+        // TODO: Depending on the method, metadata and response bytes, pick a different number of credits used
+        // This can be a slightly more complex function as we ll
+        // TODO: Here, let's implement the formula
+        let credits_used = Self::compute_cost(request_bytes, response_bytes, backend_requests == 0);
```

Comment:
this will need the method, too

Reply:
I'll add the argument to the function signature, but for now the arithmetic does not depend on method (method is not used for calculation). Lmk if this should change, easily changeable later.